frontend/packages/helm-plugin/console-extensions.json

```
@@ -1,4 +1,89 @@
 [
+  {
+    "type": "console.resource-metadata",
+    "properties": {
+      "model": {
+        "group": "helm.openshift.io",
+        "version": "v1beta1",
+        "kind": "HelmChartRepository"
+      },
+      "label": "Helm Chart Repository",
+      "labelPlural": "Helm Chart Repositories",
```

do we need i18n support here?

```suggestion
      "label": "%helm~Helm Chart Repository%",
      "labelPlural": "%helm~Helm Chart Repositories%",
```

---

Do we actually translate our models?

---

Done.